jsonsl.c

```diff
@@ -378,7 +391,8 @@ jsonsl_feed(jsonsl_t jsn, const jsonsl_char_t *bytes, size_t nbytes)
                 goto GT_SPECIAL_NUMERIC;
             }
 
-            if (state->special_flags & JSONSL_SPECIALf_NUMERIC) {
+            if (state->special_flags & JSONSL_SPECIALf_NUMERIC &&
```

Comment:
I've added the parens. I thought about ifdef'ing but I agree that the compiler will optimize this out, and it keeps the code much cleaner and easier to work on if there are fewer ifdefs.